src/main/java/net/minecraftforge/fluids/FluidAttributes.java

```diff
@@ -380,6 +530,12 @@ public final Builder color(int color)
         public final Builder overlay(ResourceLocation texture)
         {
             overlayTexture = texture;
+            return viewOverlay(texture);
+        }
+
+        public final Builder viewOverlay(ResourceLocation texture)
+        {
+            viewOverlayTexture = new ResourceLocation(texture.getNamespace(), "textures/" + texture.getPath() + ".png");
```

Comment:
Are there any cases where someone may wish to set an overlay but not the viewOverlay? (Not fully sure how the two overlays interact/work)